### DISCUSSION TITLE
Remote training related issues

### DISCUSSION THREAD

**Discussion by htaoruan** [open-mmlab/mmocr#1804] [General]
Hi, my program is running on a remote server and when I close the terminal like VS, the program will not run, so I use the following command:
nohup CUDA_VISIBLE_DEVICES=0 python tools/train.py configs/textdet/dbnet/dbnet_resnet50-dcnv2_fpnc_1200e_icdar2015.py --work-dir dbnet/ - -amp
Then there will be an error, so how should I get the program to run on the back end?

**Comment by Harold-lkk** [comment]
The easy way is to use `tmux`
